sentences/de/homeassistant_HassCancelAllTimers.yaml

```diff
@@ -0,0 +1,7 @@
+language: de
+intents:
+  HassCancelAllTimers:
+    data:
+      - sentences:
+          - "<timer_cancel> alle[ meine] Timer"
+          - "<timer_cancel> alle[ meine] Timer <area>"
```

Comment:
Would it make sense to use the `<alle>` expansion rule here?
This would allow `stoppe die ganzen Timer im Wohnzimmer` or `stoppe sämtliche Timer im Wohnzimmer`